packages/scoutgame/src/tokens/getClaimableTokens.ts

```diff
@@ -41,10 +43,10 @@ export async function getClaimablePoints({
     }
   });
 
-  const totalUnclaimedPoints = pointsReceipts.reduce((acc, receipt) => acc + receipt.value, 0);
+  const totalUnclaimedTokens = tokensReceipts.reduce(
+    (acc, receipt) => acc + Number(formatUnits(BigInt(receipt.value), 18)),
```

Comment:
nice-to-have: its best to import devTokenDecimals instead of hard-coding 18... its a nice reference also that these tokens are dev tokens.